src/components/ProseApiReference.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+  <div
+    class="
+      prose prose-slate max-w-none
+      prose-h2:mt-4 prose-h2:font-bold prose-h2:type-headline-4
+      prose-h3:scroll-mt-20 prose-h3:type-subtitle-1 prose-h3:font-bold
+      prose-lead:text-slate-500
+      prose-a:no-underline prose-a:text-content-primary hover:prose-a:no-underline
+      prose-img:mb-8 prose-img:mx-auto prose-img:object-contain
```

Comment:
Question: Do we have any images in our API reference? This may not be needed.

Reply:
Yep we have a couple of images e.g. https://docs.centrapay.com/api/asset-transfers